addons/hr_holidays/models/resource.py

```diff
@@ -157,6 +160,57 @@ def unlink(self):
 
         return res
 
+    @api.model
+    def load_public_holidays(self, companies=[], from_ui=False):
```

Comment:
easier to default `from_ui` to True and enforce False in the post init call.
Although, is it correct to avoid to localise when this method is called at module install? I feel like the server might very well be localised in a specific timezone as well :thinking: Not sure about that though, but if so the argument isn't necessary and the date should be localised anyway :slightly_smiling_face: 

Additionally, the variable names are inverted, put `_ids` on ids list and not recordsets
```suggestion
    def load_public_holidays(self, company_ids=[]):
```